README.md

```diff
@@ -23,6 +23,17 @@ required to run `ursula`:
 * libxslt-dev
 * libffi-dev
 
+### RHEL System Dependencies
+
+The following packages are needed on Red Hat Enterprise Linux environments:
+
+* python-pip
+* python-devel
+* python-virtualenvwrapper
+* libxml2-devel
+* libxslt-devel
+* libffi-devel
+
```

Comment:
Would suggest for all deployments that we recommend updating pip and virtualenv:
```pip install -U pip virtualenv```